tests/Schema/ComparatorMySQLTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Schema;
+
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Column;
+use Doctrine\DBAL\Schema\ColumnDiff;
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\SchemaDiff;
+use Doctrine\DBAL\Schema\TableDiff;
+use Doctrine\DBAL\Types\Type;
+
+class ComparatorMySQLTest extends ComparatorTest
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->platform = new MySQLPlatform();
+    }
+
+    public function testCompareChangedBlobColumn(): void
```

Comment:
How is this test different from `testAlterTableChangeBlobColumnLength()`?

Reply:
It checks if the changedProperties get reported correctly. But as we now want to change this to only return `true` or `false` I could remove the `testAlterTableChangeBlobColumnLength()` from the platform test I think.

Reply:
Fixed now as the other tests got removed.